sentence_transformers/sampler.py

```diff
@@ -77,21 +97,17 @@ def __init__(
         dataset: Dataset,
         batch_size: int,
         drop_last: bool,
-        valid_label_columns: list[str] = None,
-        generator: torch.Generator = None,
-        seed: int = 0,
```

Comment:
I believe there was an issue with the typing and usage of the seed parameter, given that 0 is falsy, it would be ignored if it was passed as a seed.